doc/Getting-Started.org

```diff
@@ -215,3 +215,24 @@ conexp.fca.implications/parallel-canonical-base-from-clop
   predicate.
 nil
 #+end_src
+
+** Console Client
+
+~fca-clj~ is a standalone console application that supports some of ~conexp-clj~ function and can conveniently be run from the command line.
+It can be downloaded here:	[[https://github.com/jana-fischer/fca-clj]]
```

Comment:
Link should probably point to `https://github.com/tomhanika/conexp-clj-command-line-tool`?